Classes/TMDebugConsole.m

```diff
@@ -185,31 +185,17 @@ - (void)setUpDateFormatter
 
 - (void)logMessage:(DDLogMessage *)logMessage
 {
-	logMessage = [self formatLogMessage:logMessage];
-	[self scheduleLogMessage:logMessage];
-}
-
-- (DDLogMessage *)formatLogMessage:(DDLogMessage *)logMessage
-{
-	if (formatter)
-	{
-		NSString *logMsg = [formatter formatLogMessage:logMessage];
-		if (logMsg != nil)
-		{
-			logMessage->logMsg = logMsg;
-		}
-		else
-		{
-			logMessage = nil;
-		}
-	}
-	return logMessage;
+    NSString *logMsg = [formatter formatLogMessage:logMessage];
+    
+    if (logMsg)
+    {
+        logMessage->logMsg = logMsg;
```

Comment:
Or could you leave the code as it was before and apply the format in [TMDebugConsole:textForMessage:](https://github.com/cesteban/TMDebugConsole/blob/master/Classes/TMDebugConsole.m#L271)?